README.md

```diff
@@ -7,17 +7,37 @@ A list of awesome mobbing/ensemble/team programming resources. PRs are very welc
 ### Web
 
  - [mobti.me](https://mobti.me) - A free, collaborative, real-time mobbing timer.
+ - [mob.sh](https://mob.sh) - A free, script which gives you a simple set of shell commands for a fast handover.
+ - [timer.mob.sh](https://timer.mob.sh) - A timer, for the mob.sh script.
+ - [Mob.sh Intellij Plugin](https://plugins.jetbrains.com/plugin/14266-mob) - A plugin version of mob.sh.
 
 ## Social Media
 
 ### Twitter accounts
 
- - [@mobtime_app](https://twitter.com/mobtime_app)
+- [@mobtime_app](https://twitter.com/mobtime_app)
+- [@mob__mentality](https://twitter.com/mob__mentality)
+- [@allenholub](https://twitter.com/allenholub) 
 
 ## Companies who Mob
 
  - [vehikl](https://vehikl.com) - We're a software consultancy specialising in PHP and JavaScript. We build best-in-class web applications written with popular frameworks such as Laravel, Express, NestJS, Vue, and React. Think of us as an extension of your team.
+ - [INNOQ](https://www.innoq.com/de/podcast/061-remote-mob-programming/) - German Company which supports mob programming
 
+## Literature
+
+ - [Mob Programming: A whole Team approach](https://leanpub.com/mobprogramming) - Great book about the beginning of Mob Programming
+ - [Remote Mob Programming](https://leanpub.com/remotemobprogramming) - A short book about remote Mob Programming
+ - [Code with the Wisdom of the Crowd: Get Better Together with Mob Programming](https://amzn.to/3JK0vt9) - A short book about Mob Programming
+
+## Podcasts
+
+### Mob Mentality Show
```

Comment:
Let's turn this into

- Mob Mentality Show
  - Apple
  - Google
  - Spotify
  - Amazon

Reply:
looks better than my approach. I will change it